packages/playground/src/utils/delete_deployment.ts

```diff
@@ -68,6 +68,9 @@ export async function deleteGateway(grid: GridClient, name: string) {
   const { deleted } = await grid.gateway.delete_name({ name });
   return deleted.length > 0;
 }
+export async function deleteGatewayContract(grid: GridClient, name: string) {
```

Comment:
this method deletes a gateway deployment. it deletes the name and node contract for this gateway deployment.
```suggestion
export async function deleteGatewayDeployment(grid: GridClient, name: string) {
```